code/src/App.js

```diff
@@ -1,9 +1,83 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
+
+
+import MessageForm from './components/MessageForm'
+import MessageList from './components/MessageList'
+import { API_URL, LIKES_URL } from './reusable/urls'
 
 export const App = () => {
+  const [messageList, setMessageList] = useState([]);
+  const [messageNew, setMessageNew] = useState('');
+
+
+  useEffect(() => {
+    fetchMessageList();
+  }, []);
+
+  const fetchMessageList = () => {
+    fetch(API_URL)
+    .then(res => res.json())
+    .then(messages => setMessageList(messages))
+    .catch(err => console.error(err));
+  }
+
+  const handleMessageNewChange = (event) => {
+    setMessageNew(event.target.value);
+  }
+
+  const handleFormSubmit = (event) => {
+    event.preventDefault();
+    event.target.reset();
+
```

Comment:
This comment is probably more for me but how on earth I didn't manage to get this one line to work in my code. 😆